### ISSUE TITLE
Can't load savegames

### ISSUE THREAD

**Issue by Joe141414** [d07RiV/diabloweb#126]
I have the Diabdat.mpq file so I can actually play the whole version of the Diablo web port. But when I save my singleplayer game (single_0.sv) I can't find any way to tell the web port of Diablo to load it again the next day. When I go to "Manage Saves" before starting the game, it seems to be doing nothing. Am I doing something wrong, or is there a bug with my browser (I use Chrome).

**Comment by Bumbar14** [comment]
Do you have solution? I have same issue.

**Comment by AJenbo** [comment]
you load it drom the main menu od the game

**Comment by Bumbar14** [comment]
When I start game I do not have load game available in main menu. Could it be due to laptopm rconfiguration and on restart it deletes this data? (I am using companys computer which is incredibly locked). So if it save it anywhere on local computer, can you please give me location so that I can copy this file, make backup and then next day past it where it should be?

**Comment by AJenbo** [comment]
Save games are in localstorage, so if that is cleared the save game will be lost unless you use the save game manger to copy the game to a safe location, then later you can drag and drop the save game to the website and it should then appear in the list of save games.

**Comment by Bumbar14** [comment]
Great. I know where to look. What is name of save game file? Because I looked and could not find it. Mybe I was not looking at correct location (using Chrome, win 10 - location was C:\Users\XXXXX\AppData\Local\Google\Chrome\User Data\Default\Local Storage.

**Comment by AJenbo** [comment]
I don't think it will appear as files on your file system, if it's not i the save game list then it's already gone from local storage. The files should be called single_#.sv

**Comment by Bumbar14** [comment]
OK. Then I will use save game manager. Just a quick question how to use it. After I save game I reload a page and save game manager appears and then I can download this file?

**Comment by AJenbo** [comment]
correct

**Comment by user95f85f** [comment]
I'm using Chrome web browser and when I upload my save my diablo game is 0%able to detect the file;
I'll try updating my Chrome browser;

**Comment by AJenbo** [comment]
You could also use DevilutionX for a native Linux build.
